webpack.config.ts

```diff
@@ -86,7 +87,7 @@ const config: WebpackConfiguration & {
     new EnvironmentPlugin({
       DATA_SOURCE: 'remote',
       BRAND_TYPE: 'Konveyor',
-      NAMESPACE: 'konveyor-forklift',
+      NAMESPACE: 'openshift-mtv',
```

Comment:
ehhh, ok, we can do downstream :-)

Reply:
really this should come from and envvar or similar -- it is only in this change because I thought I reset the file before changing it for this PR, but I didn't actually do that.

todo list!